giant_news/tests/test_feeds.py

```diff
@@ -0,0 +1,78 @@
+from django.test import TestCase, override_settings
+from django.urls import reverse
+from django.utils import timezone
+from django.utils.feedgenerator import Atom1Feed
+
+from giant_news import models
+from giant_news import feeds
+
+
+@override_settings(NEWS_FEED_TITLE="Giant News RSS Feed")
+@override_settings(NEWS_FEED_DESCRIPTION="Latest articles from Giant News")
+class RSSFeedTestCase(TestCase):
```

Comment:
Ideally these would be in "pytest syntax" to keep consistency with the rest of the tests

Reply:
The test class is a Django TestCase. I couldn't figure out how the pytest fixtures worked in the other tests. I'd happily be corrected.